src/test/resources/data/recipe_book_extension_test/recipes/recipe_book_extension_recipe_4.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Since the mod can be enabled/disabled, this should be using the conditional recipe system.